packages/salesforcedx-apex-debugger/package.json

```diff
@@ -23,7 +23,7 @@
     "@types/async-lock": "0.0.20",
     "@types/chai": "^4.0.0",
     "@types/mocha": "2.2.38",
-    "@types/node": "8.9.3",
+    "@types/node": "10.1.4",
```

Comment:
@vazexqi want to point out this last change before I merge this change. I didn't see any code changes needed after updating to this types/node version.

Reply:
FYI: we synced offline and decided to make this change later as a separate PR since we don't want to encourage usage of node 10 api's just yet.